--- CMakeLists.txt	2020-12-24 09:36:46.692170595 +0200
+++ CMakeLists.txt	2020-12-24 09:08:26.490494596 +0200
@@ -2,7 +2,7 @@
 project(iot-embeddedc)
 enable_testing()
 
-option (run_tests "set run_tests to ON if build tests should be run, set to OFF to skip tests" ON)
+option (run_tests "set run_tests to ON if build tests should be run, set to OFF to skip tests" OFF)
 SET(CMAKE_CXX_FLAGS "-g -O0 -Wall -fprofile-arcs -ftest-coverage -fPIC ${CMAKE_CXX_FLAGS} ")
 SET(CMAKE_C_FLAGS "-g -O0 -Wall -W -fprofile-arcs -ftest-coverage -fPIC ${CMAKE_C_FLAGS} -I/usr/local/include")
 SET(CMAKE_EXE_LINKER_FLAGS "-fprofile-arcs -ftest-coverage ${CMAKE_EXE_LINKER_FLAGS} -L/usr/local/lib")
